sources/packages/web/src/components/generic/ToggleContent.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <div v-if="toggled">
-    <v-row justify="center" class="mt-4">
+    <v-row justify="center" class="mt-1">
       <slot name="image">
         <v-img
-          height="200"
+          height="150"
```

Comment:
why did we reduce the height here?

Reply:
I was on a call with @hellolynn-tbtb, we used the Kitten for the "no-location", the "empty template" was big and we talked about reducing it globally. At least that was my understanding. Correct me if I am wrong @hellolynn-tbtb 😃